native/react-native/rn-by-example/basicLayoutClass.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * ES6 gives us access to imports! You can still use require(), and you may want to, when
+ * appropriate.  Reserve this for situations where you don't need all your imports on load
+ * and require them inline when they're needed. Otherwise, just use import and put all of
+ * them at the top of your file so they're easy to find.
+ */
+
+import React, { Component } from "react";
+import { Text, View } from "react-native";
+
+/**
+ * Okay...  What's up with the braces? Even if you didn't ask that, I'm telling you anyhow
+ * because it's important. The difference between the ones with the braces and the ones
+ * without is this: If the component is exported by name in the module's index.js file,
+ * that means you can pick and choose which parts of the module you want. You may only
+ * want one component or utility from a library that has like 100. In our class here,
+ * I'm importing Text and View by name from the react-native library. Some components
+ * offer this and some don't.  Always check the API before coding.
+ */
+
+class BasicLayoutClass extends Component {
+    /**
+     * This is a class component. When you add the "class" directive and extend Component,
+     * you can make your own Component. Since this component is a class, we can give it
+     * some state!  Functional components, by contrast, cannot have state.  Let's give
+     * this class some initial state.  We do that in the constructor.
+     */
+
+    constructor(props){
```

Comment:
I remember reading an article that I think @j-ornelas sent me about not needing to use the constructor any more. Is it something we should still be using?